### PR TITLE
Chore - make deploy script more configurable

### PR DESCRIPTION
**Pull request by ilinzweilin**

### CODE REVIEW

File: bin/deploy.sh [hunk 1]
```diff
@@ -30,21 +30,11 @@ source ./borrower/deploy.sh
 
 # finalize deployment
 message Finalize Deployment
-GOVERNANCE="0x$ETH_FROM"
 
 seth send $ROOT_CONTRACT 'prepare(address,address,address)' $LENDER_DEPLOYER $BORROWER_DEPLOYER $GOVERNANCE
```
Comment (manuelpolzhofer) [inline bin/deploy.sh]: if you remove it this line would not work anymore?

File: bin/deploy.sh [hunk 2]
```diff
@@ -30,21 +30,11 @@ source ./borrower/deploy.sh
...
 seth send $ROOT_CONTRACT 'deploy()'
 
 success_msg "Tinlake Deployment Finished"
 success_msg "Deployment File: $(realpath $DEPLOYMENT_FILE)"
 
-#touch $DEPLOYMENT_FILE
-addValuesToFile $DEPLOYMENT_FILE <<EOF
```
Comment (manuelpolzhofer) [inline bin/deploy.sh]: I think we should add it the deployment json file to know for each deployment who is in control

File: bin/deploy.sh (hunk 1 above)
Comment (ilinzweilin) [reply bin/deploy.sh]: it will, I have added it to the local config

File: bin/deploy.sh (hunk 2 above)
Comment (ilinzweilin) [reply bin/deploy.sh]: it is added, but already in the setup_local_config_file

**Pull request merged**